opm/material/common/UniformXTabulated2DFunction.hpp

```diff
@@ -310,12 +345,14 @@ class UniformXTabulated2DFunction
     {
         if (xPos_.empty() || xPos_.back() < nextX) {
             xPos_.push_back(nextX);
+            yPos_.resize(xPos_.size());
```

Comment:
why is `.resize()` used here, while the code below uses `.insert()`?

Reply:
To match the already existing pattern for samples_, otherwise push_back() would do. I think using insert() communicates a potential performance hit, so I prefer to avoid it when there are alternatives.

Reply:
I think using `push_back()` would be better here. the performance should actually be identical because both methods must re-allocate the memory and copy the old contents if the array capacity is exceeded.

Reply:
Sure. I will then also change samples_ to be modified with push_back() instead of resize.